Makefile

```diff
@@ -10,6 +11,10 @@ default:
 check:
 	TOXENV=py38 tox
 
+.PHONY: format
+format:
+	TOXENV=format tox
```

Comment:
Can we just add this to the "lint" target? In my mind they're basically the same thing.

Reply:
Yes we could do that. Contributors would just run `make lint` to format the code and check it.

Reply:
Good idea, that would probably be the best of both worlds!